facebook.js

```diff
@@ -78,10 +78,11 @@ const emoji_map = {
   "wedding": "💒"
 }
 
-const show_facebook_cv_tags = function() {
+const show_facebook_cv_tags = function(obj) {
   const TAG_PREFIX = "Image may contain: ";
-  const images = [...document.getElementsByTagName('img')];
-
+  let images = [...obj.querySelectorAll('img.img')];
```

Comment:
is better than selecting all the imgs in the DOM even if they are not photos, and they are a lot.
Anyway could be a bit slow (I haven't noticed btw) on the first load, the this is applied only on the target returned from the mutation